server/src/main/java/io/druid/initialization/Initialization.java

```diff
@@ -142,6 +142,8 @@ static void clearLoadedModules()
               "Extension module [%s] was ignored because it doesn't have a canonical name, is it a local or anonymous class?",
               module.getClass().getName()
           );
+        } else if (config.getExcludeModules().contains(moduleName)) {
```

Comment:
would it make sense to extend this functionality with an ability to exclude modules on the package level (or by regex) instead of explicitly enumerate all of them by class name?